lib/jira/commands/tickets.rb

```diff
@@ -0,0 +1,22 @@
+module Jira
+  class CLI < Thor
+
+    desc "tickets", "List the tickets of the input username"
```

Comment:
It looks like JIRA is returning all the tickets ever assigned to the user. For example, I have 2.6k tickets that have ever been assigned to me. Happy to solve this issue later, but maybe add a comment with a todo for solving the pagination issue